daprdocs/content/en/reference/cli/dapr-publish.md

```diff
@@ -19,7 +19,8 @@ dapr publish [flags]
 
 | Name | Environment Variable | Default | Description
 | --- | --- | --- | --- |
+| `--publish-app-id` | `-i`| | The id for publishing application
```

Comment:
@artursouza Is this the id of the application you're publishing to or a separate id?

Reply:
A separate ID, its the ID of the application you are using to publish.

Reply:
Any recommendation on the wording or is it clear?

Reply:
Maybe "An ID that represents the app from which you are publishing"?

Reply:
Done.